### PR TITLE
PMM-12921 added new fb checks job

### PR DESCRIPTION
**Pull request by vyarosh**
https://jira.percona.com/browse/PMM-12921

### CODE REVIEW

**Comment by vyarosh** [comment]
dev-latest: https://github.com/Percona-Lab/pmm-submodules/actions/runs/8057254229

File: .github/workflows/pmm-qa-fb-checks.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+name: "Feature Build: pmm-qa checks"
```
Comment (yurkovychv) [inline .github/workflows/pmm-qa-fb-checks.yml]: Can we change the name to a shorter one?

File: .github/workflows/pmm-qa-fb-checks.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    secrets:
+      BACKUP_LOCATION_ACCESS_KEY:
+        required: false
+      BACKUP_LOCATION_SECRET_KEY:
+        required: false
+
+  push:
+    branches:
+      - PMM-12921-pmm-qa-checks
```
Comment (yurkovychv) [inline .github/workflows/pmm-qa-fb-checks.yml]: this event should be also removed I think

File: .github/workflows/pmm-qa-fb-checks.yml (hunk 1 above)
Comment (vyarosh) [reply .github/workflows/pmm-qa-fb-checks.yml]: Sure, which name would you prefer?

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**